tutorials/plot_ica_from_raw.py

```diff
@@ -1,8 +1,8 @@
 """
 .. _tut_preprocessing_ica:
 
-Compute ICA on MEG data and remove artifacts
-============================================
+Example: Compute ICA on MEG data and remove artifacts
```

Comment:
I think most of our examples don't start wiht "Example:", right? I think it's cleaner probably not to have it there